content/klc/S2.1.adoc

```diff
@@ -2,15 +2,15 @@
 brief = "General symbol naming guidelines"
 +++
 
-. Library name should not be duplicated in symbol name
+. Library naming should not be duplicated in footprint name
 . If symbol with same name exists for multiple manufacturers, the manufacturer name is written first
-. Specific manufacturer name (for atomic parts)
-. Type of symbol (for generic parts)
+. Fully specified symbols are named based on the manufacturer part number
+.. If and only if multiple manufacturers build compatible devices then the common base part number is used with clear name suffix for the footprint (e.g. `L78L05_TO92`)
+.. Non functional parts of the MPN are replace with a wildcard. (Refer to link:/libraries/klc/S2.2[requirements for the use of wildcards in MPN])
```

Comment:
'Non-functional' is missing the hyphen. 'replace' should be past tense or the entire sentence re-worked to change tense.

Reply:
done